test/performance/odp_l2fwd.c

```diff
@@ -173,9 +188,26 @@ typedef union ODP_ALIGNED_CACHE {
 	uint8_t padding[ODP_CACHE_LINE_SIZE];
 } stats_t;
 
+/* Transmit completion specific state data */
+typedef struct {
+	odp_packet_tx_compl_opt_t opt;
+	uint32_t init;
+	uint32_t max;
+	uint32_t free;
+	uint32_t compl;
+	int ival;
+	int n;
```

Comment:
Comments in these fields would help in understanding the code.

Reply:
Comments added and variable names improved in v2.